### ISSUE TITLE
Framing: publishing the NRRD vision & framing on the wiki

### ISSUE THREAD

**Issue by alexpandel** [18F/doi-extractives-data#2636]
Let's get out product framing out of an 18F-owned Google Doc and into the project's [Github wiki](https://github.com/18F/doi-extractives-data/wiki) so we can share it more easily with stakeholders, etc!

This is done when the following pieces of our framing doc are reflected in the wiki:
- [x] the problem
- [x] our vision 
- [x] prioritized user scenarios & user promises
- [x] what we're not trying to do
- [x] project risks
- [x] reorganize to archive historical stuff

**Comment by alexpandel** [comment]
@brentryanjohnson @jennmalcolm would you two be up for taking a stab at this next week? Pulling it into this sprint since it's basically just a continuation of #2551 :)

**Comment by brentryanjohnson** [comment]
I quickly added [our product framing docs](https://github.com/18F/doi-extractives-data/wiki) to the wiki today. They could use a review by @jennmalcolm and some clean up to the tables and formatting.

**Comment by alexpandel** [comment]
Yayyy to having this in a more public place! 🎉Two thoughts:

1) I wonder if it might make more sense to move the older problem statements and hypotheses into more of an archive section, and keep all the more recent stuff together in one place? Right now it's a little hard to parse which pieces are still relevant and which are much older/out of date. Relatedly, I wonder if we'd want to rename the "process" section something more like "product strategy" or "product direction" since it's not as much about "process" anymore? @jennmalcolm / @coreycaitlin, any thoughts here?
2) I see what you mean re: table formatting. Hmmm. For the risks table, I think you should be able to add bullets inside the "how we might mitigate" column to help it be a little more legible? But as for the user scenarios one... yeah, that one is pretty tough to read. Have any ideas for how we might present that one a little differently? I wonder if something like the following could work?:

**P1: [insert user scenario here]**
- P1: [insert related product goal here]
- P1: [insert related product goal here]
- P2: [insert related product goal here]

**P2: [insert user scenario here]**
- P2: [insert related product goal here]
- P2: [insert related product goal here]
- P3: [insert related product goal here]

**Comment by coreycaitlin** [comment]
Any objections to starting a "history" section/page? that could give us a spot to move things that we don't really need in the active section, but that might be handy for future reference.

**Comment by alexpandel** [comment]
I like that idea, @coreycaitlin! Relatedly, in drafting up the weekly update email today and wanting to link to one place that represents all our product framing... it might be nice to have all the framing elements represented on a single wiki page, vs spread out across several pages, to make it easier to link to as one cohesive doc? I recognize that means it would be quite long, but we could always link to individual sections on the page as needed?

**Comment by brentryanjohnson** [comment]
@alexpandel I struggled with how to organize it...I thought it might be useful to have individual pages for each, assuming we would want to archive the iterations within the same context. But if we move the past versions into an archive, having the framing elements together on one page makes more sense.
